build/org.eclipse.pde.build/src/org/eclipse/pde/internal/build/site/PDEState.java

```diff
@@ -441,9 +441,11 @@ public void resolveState() {
 					IExecutionEnvironment env = JavaRuntime.getExecutionEnvironmentsManager().getEnvironment(javaProfiles[j]);
 					String systemPackages = getSystemPackages(env, profileProps);
 					String ee = profileProps.getProperty(FRAMEWORK_EXECUTIONENVIRONMENT);
-
+                                         
```

Comment:
Please amend the commit to avoid adding whitespace.

Reply:
done

Reply:
Looks fine now.  I'm not sure if folks will insistent that you squash this to a single commit.

I'll leave that to @HannesWell to consider...